new-tab-page/new-tab-page-api/src/main/java/com/duckduckgo/newtabpage/api/NewTabPageSectionProvider.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.newtabpage.api
+
+import kotlinx.coroutines.flow.Flow
+
+interface NewTabPageSectionProvider {
```

Comment:
Needs documentation

Reply:
Added task to take care of it once the API is stable https://app.asana.com/0/1174433894299346/1207701719136415/f